net-p2p/mldonkey/mldonkey-3.1.7.ebuild

```diff
@@ -0,0 +1,187 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit flag-o-matic eutils autotools toolchain-funcs
+
+IUSE="bittorrent doc fasttrack gd gnutella gtk guionly magic +ocamlopt upnp"
+
+DESCRIPTION="Multi-network P2P application written in Ocaml, with Gtk, web & telnet interface"
+HOMEPAGE="http://mldonkey.sourceforge.net/"
+SRC_URI="https://github.com/ygrek/mldonkey/releases/download/release-${PV//./-}-2/${P}-2.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND="dev-lang/perl
+	dev-ml/camlp4:=
+	guionly? ( >=gnome-base/librsvg-2.4.0
+			>=dev-ml/lablgtk-2.6 )
+	gtk? ( >=gnome-base/librsvg-2.4.0
+			>=dev-ml/lablgtk-2.6[svg] )
+	gd? ( >=media-libs/gd-2.0.28[truetype] )
+	magic? ( sys-apps/file )
+	upnp? ( net-libs/miniupnpc:= net-libs/libnatpmp:= )
+"
+
+DEPEND="${RDEPEND}
+	acct-user/p2p
+	<dev-lang/ocaml-4.10[ocamlopt?]
+	bittorrent? (
+		|| (
+			<dev-lang/ocaml-4.06
+			(
+				>=dev-lang/ocaml-4.06
+				dev-ml/num
+			)
+		)
+	)"
+
+RESTRICT="!ocamlopt? ( strip )"
+
+MLUSER="p2p"
```

Comment:
What does this line do?

Reply:
Yeah, that should be removed. It was used in previous ebuils when calling `enewuser`:

```bash
enewuser ${MLUSER} -1 -1 /home/p2p users
```

But I moved that to `acct-user/p2p` and this is a stale line that I must remove.